### PR TITLE
win_firewall_rule: fix enable syntax

### PR DESCRIPTION
**Pull request by ilpianista**
The `enable` parameter is a string, then compare it with strings. I suppose this
never worked before.

While I'm on it, I also fixed a typo in the example and added a required
parameter.

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks @ascarpino. @TimothyVandenbrande please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit' or 'needs_revision' as appropriate.

**Comment by t-v** [comment]
curious, since I am still using it in that state, on a weekly basis (ansible 1.9.2), but this one still works, so shipit.

**Comment by gregdek** [comment]
Thanks again to @ilpianista for this PR, and thanks @TimothyVandenbrande for reviewing. Marking for inclusion.

File: windows/win_firewall_rule.ps1 [hunk 1]
```diff
@@ -200,7 +200,7 @@ $fwsettings=@{}
 # Variabelise the arguments
 $params=Parse-Args $args;
 
-$enable=Get-Attr $params "enable" $null;
+$enable=Get-Attr $params "enable" "";
```
Comment (bcoca) [inline windows/win_firewall_rule.ps1]: If enable is a boolean it should default to False or at least null, not empty string, also this does not reflect what the docs state as the default.

**Comment by ilpianista** [comment]
Hi @bcoca, what the docs state as default? Because I see it says ['yes', 'no'], not true/false.

**Comment by bcoca** [comment]
default is 'null', yes/no are equivalent to true/false (also on/off, 0/1, they are all YAML/Ansible booleans)

**Comment by gregdek** [comment]
@ilpianista A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes.

**Comment by ilpianista** [comment]
@gregdek thanks, I'll try to review it ASAP.

**Comment by ilpianista** [comment]
So, this is what I get when I use the following role with ansible 2.0.1.0:

```
- name: Open required port on the firewall
  win_firewall_rule: name="test_8080" localport=8080 protocol=TCP action=allow direction=In profile=public enable=yes
```

Result:

```
failed: [test] => (item=8080) => {"changed": false, "failed": true, "item": "8080", "msg": ["for the enable parameter only yes and no is allowed"]}
```

While it works when I use `enable=true`. Documentation says to use `yes/no`.

**Comment by gregdek** [comment]
@ilpianista Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If we don't hear from you within another 14 days, we will close this pull request.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by ilpianista** [comment]
This PR needs_revision, no actions from my side.

**Comment by gregdek** [comment]
Putting this back into shipit.

**Pull request merged**